### PR TITLE
hidden properties

### LINKED ISSUE #1395: [🐛 BUG] Hidden element properties in viselements.json are generated by the Page Builder API
### What went wrong? 🤔

[This taipy-doc issue](https://github.com/Avaiga/taipy-doc/issues/977) provides means to hide the documentation of specific properties for any given element.
However, as of today, these properties still appear as valid parameters for the Page Builder API.

### Expected Behavior

Hidden properties should not be exposed to the API.

### Steps to Reproduce Issue

Starting in Taipy 3.2, the 'hover_text' property of the 'indicator' element should be hidden.
It however shows up when creating an indicator using `tpb.indicator()`.

### Solution Proposed

_No response_

### Screenshots

_No response_

### Runtime Environment

_No response_

### Browsers

_No response_

### OS

_No response_

### Version of Taipy

3.2

### Additional Context

_No response_

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
password reveal icon
pass type
resolves #1395

### CODE REVIEW

**Review by namnguyen20999 — DISMISSED** [review]
Looking good !!!!

File: taipy/gui/viselements.json [hunk 1]
```diff
@@ -109,6 +109,12 @@
                         "type": "int",
                         "default_value": "5",
                         "doc": "The height of the displayed element if multiline is True."
+                    },
+                    {
+                        "name": "type",
+                        "type": "str",
+                        "default_value": "text",
+                        "doc": "The type of input: text, tel, email ..."
```
Comment (FabienLelaquais) [inline taipy/gui/viselements.json]: Is there a list to pick meaningful values from?

File: taipy/gui/viselements.json [hunk 2]
```diff
@@ -109,6 +109,12 @@
                         "type": "int",
                         "default_value": "5",
                         "doc": "The height of the displayed element if multiline is True."
+                    },
+                    {
+                        "name": "type",
+                        "type": "str",
+                        "default_value": "text",
```
Comment (FabienLelaquais) [inline taipy/gui/viselements.json]:
```suggestion
                        "default_value": "\"text\"",
```

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
Nice!
A doc issue I suspect, though

File: taipy/gui/viselements.json (hunk 2 above)
Comment (FredLL-Avaiga) [reply taipy/gui/viselements.json]: good catch tx

File: taipy/gui/viselements.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply taipy/gui/viselements.json]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types

Comment (FabienLelaquais) [reply taipy/gui/viselements.json]: How do users know that?
Oh I know. Let them know.

**Review by FabienLelaquais — APPROVED** [review]
Oh that TODO did help hu?
Well done!

File: taipy/gui/viselements.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply taipy/gui/viselements.json]: or not ...

**Pull request merged**